### PR TITLE
Change to more detailed document and fix typos

### PR DESCRIPTION
**Pull request by MichaelKim20**

### CODE REVIEW

**Review by TrustHenry — APPROVED** [review]

File: source/agora/cli/client/Result.d [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************
 
-    The Agora client Result Constans
+    The Agora client Result Constant
```
Comment (Geod24) [inline source/agora/cli/client/Result.d]:
```suggestion
    Constants returned by the agora CLI client
```

Comment (Geod24) [reply source/agora/cli/client/Result.d]: Although honestly I'd rather get rid of this module, I don't think a module with 3 `int` constant makes sense.

File: source/agora/consensus/UTXOSet.d [hunk 2]
```diff
@@ -114,7 +114,7 @@ public class UTXOSet
         get an UTXOSetValue in the UTXO set.
 
         Params:
-            hash = the hash of transation
+            hash = the hash of transaction
```
Comment (Geod24) [inline source/agora/consensus/UTXOSet.d]:
```suggestion
            hash = the hash of the transaction introducing the `Output`
```

File: source/agora/consensus/UTXOSet.d [hunk 3]
```diff
@@ -171,7 +171,7 @@ public class UTXOSet
         Find an UTXOSetValue in the UTXO set.
 
         Params:
-            hash = the hash of transation
+            hash = the hash of transaction
```
Comment (Geod24) [inline source/agora/consensus/UTXOSet.d]:
```suggestion
            hash = the hash of the transaction introducing the `Output`
```

File: source/agora/consensus/data/Transaction.d [hunk 4]
```diff
@@ -26,7 +26,7 @@ import agora.common.Serializer;
 
 import std.algorithm;
 
-/// The Agora Transaction Type constans
+/// The Agora Transaction Type constant
```
Comment (Geod24) [inline source/agora/consensus/data/Transaction.d]:
```suggestion
/// Type of the transaction: defines the content that follows and the semantic of it
```

**Review by Geod24 — COMMENTED** [review]
Since we're fixing it, I rephrased it a bit more. Nice catch!

**Pull request merged**